### PR TITLE
Added Bookings Functions

### PR DESCRIPTION
**Pull request by kashing555**
Added:
Add bookings
Delete booking
Booking List

### CODE REVIEW

File: src/main/java/seedu/trackbeau/model/ModelManager.java [hunk 1]
```diff
@@ -93,11 +96,22 @@ public boolean hasCustomer(Customer customer) {
...
+
     @Override
     public void deleteCustomer(Customer target) {
         trackBeau.removeCustomer(target);
     }
 
+    @Override
+    public void addBooking(Booking booking) {
+        trackBeau.addBooking(booking);
+        updateFilteredCustomerList(PREDICATE_SHOW_ALL_CUSTOMERS);
```
Comment (flairekq) [inline src/main/java/seedu/trackbeau/model/ModelManager.java:112]: Is this supposed to be updateFilteredBookingList(PREDICATE_SHOW_ALL_BOOKINGS);?

**Review by flairekq — COMMENTED** [review]
Overall looks good just that it might have some cosmetic issues?

File: src/main/java/seedu/trackbeau/logic/commands/booking/AddBookingCommand.java [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+import seedu.trackbeau.model.Model;
+import seedu.trackbeau.model.booking.Booking;
+import seedu.trackbeau.model.customer.Customer;
+
+/**
+ * Adds a booking to trackBeau.
+ */
+public class AddBookingCommand extends Command {
+
+    public static final String COMMAND_WORD = "addbooking";
```
[the remark this answers is not in the document]
Comment (flairekq) [reply src/main/java/seedu/trackbeau/logic/commands/booking/AddBookingCommand.java:23]: Oh, wait nvm I think it is easier for us to just do it without spacing haha so addbooking is fine!

File: src/main/java/seedu/trackbeau/logic/parser/booking/AddBookingCommandParser.java [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+import seedu.trackbeau.logic.parser.ParserUtil;
+import seedu.trackbeau.logic.parser.Prefix;
+import seedu.trackbeau.logic.parser.exceptions.ParseException;
+
+/**
+ * Parses input arguments and creates a new AddCommand object
+ */
+public class AddBookingCommandParser implements Parser<AddBookingCommand> {
+    protected static final String EMPTY_SKIN_TYPE = "Skin type data not available";
+    protected static final String EMPTY_HAIR_TYPE = "Hair type data not available";
```
[the remark this answers is not in the document]
Comment (kashing555) [reply src/main/java/seedu/trackbeau/logic/parser/booking/AddBookingCommandParser.java:24]: Removed

File: src/main/java/seedu/trackbeau/model/ModelManager.java (hunk 1 above)
Comment (kashing555) [reply src/main/java/seedu/trackbeau/model/ModelManager.java:112]: Updated

File: src/main/java/seedu/trackbeau/logic/parser/ParserUtil.java [hunk 4]
```diff
@@ -152,4 +155,42 @@ public static Tag parseTag(String tag) throws ParseException {
...
+    /**
+     * Parses a {@code String startTime} into an {@code startTime}.
+     * Leading and trailing whitespaces will be trimmed.
+     *
+     * @throws ParseException if the given {@code startTime} is invalid.
+     */
+    public static LocalDateTime parseStartTime(String startTime) throws DateTimeException {
+        requireNonNull(startTime);
+        String trimmedAddress = startTime.trim();
+        DateTimeFormatter formatter = DateTimeFormatter.ofPattern("dd-MM-yyyy HH");
```
[the remark this answers is not in the document]
Comment (kashing555) [reply src/main/java/seedu/trackbeau/logic/parser/ParserUtil.java:186]: Updated

**Review by isabelteo — COMMENTED** [review]
LGTM

**Review by flairekq — COMMENTED** [review]
LGTM!

**Comment by hanhuiice** [comment]
LGTM!

**Pull request merged**